src/core/connection.h

```diff
@@ -519,6 +519,11 @@ typedef struct QUIC_CONNECTION {
     //
     QUIC_DATAGRAM Datagram;
 
+    //
+    // Local transport parameters as chosen for the handshake
+    //
+    QUIC_TRANSPORT_PARAMETERS LocalTP;
```

Comment:
On the server side, we need to use the same values for a subset of the transport parameters from the handshake in the resumption ticket (and we need to restore those values). Since an app can change the settings after the handshake completes (but before they send a resumption ticket), we need a way to save the transport parameters from the handshake.
This achieves that objective.

Reply:
After further discussion and reading of the draft, I am going to instead change this to a server-only thing and make it a chunk of memory we can free when 0-RTT is done.